packages/react-batch/README.md

```diff
@@ -1,3 +1,58 @@
 # react-batch
 
-Exports [boxart](github.com/boxart/boxart)'s Batch and BatchFactory. If using more of boxart use boxart-batch instead. Tools for building html games with React.
+Exports [boxart](https://github.com/boxart/boxart)'s Batch and BatchFactory. If using more of boxart use boxart-batch instead. Tools for building html games with React.
```

Comment:
Should we be capitalizing BoxArt consistently in body copy? I was following that convention on the main ba site, but assumed it rather than asking.